app/views/shared/schools/mentors/_can_not_remove.html.erb

```diff
@@ -6,6 +6,10 @@
   <%= t(".title") %>
 </label>
 
-<p class="govuk-body">
   <%= simple_format(t(".description", mentor_name: mentor.full_name)) %>
-</p>
+
```

Comment:
Similar to what you've done for Partner Schools, the "They are currently assigned to:" should appear conditionally to if they have any placements.

Reply:
I thought it might not apply here since this page will only appear if they have current assignments?